app/models/spree/payment/processing.rb

```diff
@@ -193,6 +199,20 @@ def gateway_options
 
       private
 
+      def ready_to_process?
```

Comment:
what if we name it `validate!` instead to signal that it may raise exceptions so it's consistent with the other methods in this module? I also find it a bit more intuitive with the verb `validate` as it's something I'm used to in Rails.

Reply:
Yeah, I think adding the `!` is good because it can raise errors and not just return true/false. Updated: 50e87a023b640fdaaf1ce649aa209dba70065154